docs/storage/volume_migration.md

```diff
@@ -0,0 +1,219 @@
+
+Example:
+Original VM with a datavolume and datavolume template:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm-dv
+  name: vm-dv
+spec:
+  dataVolumeTemplates:
+  - metadata:
+      creationTimestamp: null
+      name: src-dv
+    spec:
+      pvc:
+        accessModes:
+        - ReadWriteOnce
+        volumeMode: Filesystem
+        resources:
+          requests:
+            storage: 2Gi
+        storageClassName: local
+      source:
+        registry:
+          url: docker://registry:5000/kubevirt/alpine-container-disk-demo:devel
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: vm-dv
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: datavolumedisk1
+          interfaces:
+          - masquerade: {}
+            name: default
+        resources:
+          requests:
+            memory: 128Mi
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - dataVolume:
+          name: src-dv
+        name: datavolumedisk1
+      networks:
+      - name: default
+        pod: {}
+```
+
+The datavolume `src-dv` is migrated to the pvc `dest-pvc`:
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: dest-pvc
+spec:
+  accessModes:
+  - ReadWriteMany
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+by updating the VM:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm-dv
+  name: vm-dv
+spec:
+  updateVolumesStrategy: Migration
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: vm-dv
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: datavolumedisk1
+          interfaces:
+          - masquerade: {}
+            name: default
+        resources:
+          requests:
+            memory: 128Mi
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - persistentVolumeClaim:
+          claimName: dest-pvc
+        name: datavolumedisk1
+      networks:
+      - name: default
+        pod: {}
+```
+
+
+The destination volume may be of a different type or size than the source. It is possible to migrate from and to a block volume as well as a filesystem volume.
+The destination volume should be equal to or larger than the source volume. However, the additional difference in the size of the destination volume is not instantly visible within the VM and must be manually resized because the guest is unaware of the migration.
+
+The volume migration depends on the `VolumeMigration` feature gate. To enable the feature gate, add the following to the KubeVirt CR:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+spec:
+  configuration:
+    developerConfiguration:
+      featureGates:
+        - VMLiveUpdateFeatures
+        - VolumesUpdateStrategy
+        - VolumeMigration
```

Comment:
Here, there's no `VolumeMigration`, is that correct?
https://github.com/kubevirt/user-guide/pull/805/files#diff-176c70f4872d2227af90873ef4d4bb5cae0f2709836a794179596f27ea11e5bdR2147-R2148

Reply:
Yes because the dependency tree is `VolumeMigration` depends on `VolumesUpdateStrategy` which depends on `VMLiveUpdateFeatures`.

Reply:
Ack, thanks